sls-versions/src/main/java/com/palantir/sls/versions/OrderableSlsVersion.java

```diff
@@ -85,7 +85,11 @@ public static boolean check(String coordinate) {
         return safeValueOf(coordinate).isPresent();
     }
 
+    @Override
     @JsonValue
+    @Value.Auxiliary
```

Comment:
Would removing `@JsonValue` from the `toString` here be equivalent?

Reply:
I'm not sure what you're asking - this PR _does_ remove `@JsonValue` from `toString`.

Reply:
I meant not adding a getValue override in the subclasses, since the base method could be annotated with both `@Value.Auxiliary` and `@JsonValue` unless immutables gets upset (which I suppose wouldn't be all that surprising).

Reply:
Sure we could do that. It just feels wrong to have immutables annotation in a non-immutables abstract class.

But happy to do whichever you prefer.

The tests also pass if I exclude the `@JsonValue` annotation on the subclasses. But that's probably not a good idea, since `@JsonValue` isn't `@Inherited` so I assume this is relying on some Jackson introspect that may or may not be guaranteed.